src/cljs/nr/gameboard/board.cljs

```diff
@@ -120,21 +122,22 @@
             (and (corp? card)
                  (not (faceup? card))))
         (do (when (= side card-side)
-              (if (= (:cid card) (:source @card-menu))
+              (if (and (= (:cid card) (:source @card-menu))
+                       (= (:ghost card) (:ghost @card-menu)))
                 (close-card-menu)
-                (open-card-menu (:cid card))))
+                (open-card-menu (:cid card) (:ghost card))))
```

Comment:
Do you mean `(:ghost @card-menu)`? When is `:ghost` set on a card object?

Reply:
Below, `ghost-call` sets `:ghost` to be `true`, not a card object.

Reply:
no, this is correct. We're passing whether it's the `:ghost` version of the card or the regular one into the open-card-menu function, so it can open the menu on the correct one (since they have the same cid, since they're the same card).

The `ghost-card` fn below adds that key to the ghosts